version: "VDA 2.0"
manufacturer: "Test-Manufacturer"
serial_number: "Test-AGV"